### PR TITLE
partners/astradb: Add caches to langchain-astradb

### PR DESCRIPTION
**Pull request by cbornet**

### CODE REVIEW

File: libs/community/langchain_community/cache.py [hunk 1]
```diff
@@ -1365,6 +1366,11 @@ def get_md5(input_string: str) -> str:
 ASTRA_DB_CACHE_DEFAULT_COLLECTION_NAME = "langchain_astradb_cache"
 
 
+@deprecated(
+    since="0.1.23",
```
Comment (cbornet) [inline libs/community/langchain_community/cache.py:1370]: Using the langchain-core version as was done for the Pinecone VectorStore: https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/pinecone.py#L46
but shouldn't we use the main langchain version (or community) ?

Comment (cbornet) [reply libs/community/langchain_community/cache.py]: Changed to 0.0.22 as per https://github.com/langchain-ai/langchain/pull/17732#discussion_r1500096177

**Comment by cbornet** [comment]
Fixed conflicts and rebased.

**Comment by efriis** [comment]
Package has moved! Could you reopen here? https://github.com/langchain-ai/langchain-datastax

**Pull request closed without merging**